# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/quota/OriginInfo.cpp
# Commit: f466599094b7
# Full Hash: f466599094b7e16948794a49b0571e03eb2b4df9
# Author: Serban Stanca <sstanca@mozilla.com>
# Date: 2025-06-15 13:49:48
# Description:
#   Revert "Bug 1962133 - QM: Add accessed flag to FullOriginMetadata and persist it in the origin metadata file; r=dom-storage-reviewers,hsingh" as requested by dmeehan for causing top crash in Bug 1972221.
#   
#   This reverts commit 433768e64dd71c996694e08005854756b7c45fb3.
# ==============================================================================

diff -r 2408bee8bcc4 -r f466599094b7 dom/quota/OriginInfo.cpp
--- a/dom/quota/OriginInfo.cpp	Sat Jun 14 05:48:43 2025 +0000
+++ b/dom/quota/OriginInfo.cpp	Sun Jun 15 16:46:59 2025 +0300
@@ -70,7 +70,7 @@
 OriginStateMetadata OriginInfo::LockedFlattenToOriginStateMetadata() const {
   AssertCurrentThreadOwnsQuotaMutex();
 
-  return {mAccessTime, mAccessed, mPersisted};
+  return {mAccessTime, mPersisted};
 }
 
 FullOriginMetadata OriginInfo::LockedFlattenToFullOriginMetadata() const {